docs/docs/tutorials/visualization.ipynb

```diff
@@ -270,11 +312,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Facet grid with visualizations\n",
+    "## Facet Grid with Visualizations ***#with Visualizations? Pourquoi on précise? C'est une appelation technique? Je suis pas sûre que ça soit le bon terme - genre quelque chose de plus précis serait pas Facet Grid with Visualization Panels ou juste Facet Grid ?***\n",
+    "_______________________________________________________________________________________________________________\n",
+    "\n",
+    "***#J'ai viré ce que tu avais mis parce que je suis pas fan de parler de la RemoteLazyFrame ici. Est-ce que c'est important de le mentionner pour des raisons techniques (si oui, ignore ma modif, il faudra qu'on le rebosse autrement) ? Où en gros, tu peux aussi juste faire une visualisation Facet et c'est comme pour histplot, scatterplot etc ?***\n",
+    "\n",
+    "The facet function lets you create a grid of plots that accepts a `col` and `row` argument. You can then call the `histplot`, `scatterplot` or `curveplot` functions to decide how you want to plot your data in the columns and rows of the grid. \n",
     "\n",
-    "The next thing you can do with BastionLab is create a Facet for your RemoteLazyFrame, which accepts a 'col' and 'row' argument. You can then call the three data visualization functions previously discussed through the lens of these columns and rows, that is to say, if you have a Facet with a 'row' value of class and you call my_facet.histplot(x=\"Age\", bins=15), you will see three hsitplots, one showing the age of passengers in class 1, one for passengers in class 2 and the final one for class 3.\n",
+    "For example, if you have a Facet with a `row` value of class and you call `my_facet.histplot(x=\"Age\", bins=15)`, you will see three histplots: one showing the age of passengers in class 1, one for passengers in class 2 and the final one for class 3.\n",
```

Comment:
Say
    a row value of "Pclass"
to disambiguate as class is also a technical term in programming